static/js/reports/report.js

```diff
@@ -0,0 +1,1051 @@
+//TODO: clean up
+var app = {}, browserSettings = {}, browserStorage = $.localStorage;
+
+    'use strict';
+
+	var containerprefix = 'chart-';
+	var maxdays = 3 * 31;
+	var datastorage = {};
+	var daystoshow = {};
+	
+	var targetBGdefault = {
+		"mg/dl": { low: 72, high: 180 },
+		"mmol": { low: 4, high: 10 }
+	};
+	
+	var maxInsulinValue = 0
+		, maxCarbsValue = 0;
+    var 
+          ONE_MIN_IN_MS = 60000
+        , SIX_MINS_IN_MS =  360000
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: ['SIX_MINS_IN_MS' is defined but never used.](https://www.codacy.com/app/Nightscout/cgm-remote-monitor/file/769498838/issues/source?bid=2452374&fileBranchId=2459081#l20)